index.js

```diff
@@ -212,6 +212,17 @@
     function(x) { return x != null; }
   );
 
+  //  Applicative :: TypeClass
+  var Applicative = $.TypeClass(
+    'sanctuary/Applicative',
+    function(x) {
+      return (
+        _type(x) === 'Array' ||
+        Apply.test(x) && (hasMethod('of')(x) || hasMethod('of')(x.constructor))
+      );
+    }
+  );
+
```

Comment:
Is this definition correct according to your interpretation of the [spec][1]?


[1]: https://github.com/fantasyland/fantasy-land#applicative